packages/office-ui-fabric-react/src/utilities/decorators/withContainsFocus.tsx

```diff
@@ -20,6 +20,8 @@ export function withContainsFocus<P extends { containsFocus?: boolean }, S>(Comp
 
       this._delayedSetContainsFocus = this._async.debounce(this._setContainsFocus, 20);
       this._updateComposedComponentRef = this._updateComposedComponentRef.bind(this);
+      this._handleFocus = this._handleFocus.bind(this);
```

Comment:
did you miss this one?

Reply:
I think this one is a special case because it's within a decorator, and you can't use `@autobind` on methods in a nested class within the decorator factory.

Reply:
@micahgodbolt Can't use autobind here, when I try it gives the error 'Decorators are not valid here.' I think because the class is being returned from a function and can't be declared.